### PR TITLE
PR part 1

### PR DESCRIPTION
**Pull request by skyfallwastaken**
This PR makes the LICENSE and README look better! No wiki changes unfortunately since github doesn't let you do that but I can just copy the Markdown to the main wiki when I get contributor access!

### CODE REVIEW

**Comment by skyfallwastaken** [comment]
I think there is some stuff that needs to be changed (didn't expect this to be merged but it looks fine for now, for instance Draylar should probably be added to the list too)

File: README.md [hunk 1]
```diff
@@ -1,12 +1,15 @@
-![banner](https://raw.githubusercontent.com/Hephaestus-Dev/FibLib/master/banner.png "banner")
+<img src="https://raw.githubusercontent.com/Hephaestus-Dev/FibLib/master/banner.png" width="300" height="150" alt="Banner">
+
 # FibLib ![latest version](https://img.shields.io/github/v/release/Hephaestus-Dev/FibLib)
 This is a library all about lying to your players! 
 
 ## Setup
-For setup instructions please see the [FibLib wiki](https://github.com/Hephaestus-Dev/FibLib/wiki).
+For setup instructions please see the [wiki.](https://github.com/SkyfallWasTaken/FibLib/wiki)
```
Comment (Draylar) [inline README.md]: Let's change the link to https://github.com/Haven-King/FibLib/wiki & transfer your wiki over to this repository.

**Review by Draylar — DISMISSED** [review]
Oops, forgot to follow this repository & missed the PR😆

File: README.md (hunk 1 above)
Comment (skyfallwastaken) [reply README.md]: Done

**Review by Draylar — APPROVED** [review]

**Comment by skyfallwastaken** [comment]
I merged it to the 1.17 branch since it seems it has more recent commits (5 months ago compared to 8)

**Pull request merged**